CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   be breaking but it does represent a major change in the gem's implementation
   and that is the reason for the major version bump.
 - Require Ruby 2.6 or later.
+- Test against sidekiq 6.2
```

Comment:
I don't think this needs a changelog entry. 

And now I realize that I forgot to remove `(unreleased)` from the changelog. This has actually been released. 🤦

Reply:
Pushed a commit to master to fix this.

Reply:
Removed